lib/improver/cli/combine.py

```diff
@@ -31,96 +31,30 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Script to combine netcdf data."""
 
-import warnings
-
 import iris
+from iris.cube import CubeList
 
-from improver.argparser import ArgParser
+from improver.cli.clize_routines import (clizefy, inputcube,
+                                         inputjson, outputcube)
 from improver.cube_combiner import CubeCombiner
-from improver.utilities.cli_utilities import load_json_or_none
-from improver.utilities.load import load_cube
-from improver.utilities.save import save_netcdf
-
-
-def main(argv=None):
```

Comment:
Please don't remove the `main` functions just yet - we need them for as long as they're used in the suite. We will remove them after the suite is migrated to the new CLIs/APIs. Otherwise we might have to delay merging this PR into master.